examples/spring-streams-example/src/test/kotlin/com/stove/spring/streams/example/e2e/ExampleTest.kt

```diff
@@ -57,43 +54,29 @@ class ExampleTest : FunSpec({
          ----------------------------*/
 
         //  Assert input1 message is consumed
-        shouldBeConsumed<Any> {
-          protobufSerde.messageAsBase64(actual)
-            .isSome { message ->
-              message.onMatchingAssert(Input1.getDescriptor().name) {
-                Input1.parseFrom(it.toByteArray()) == input1Message
-              }
-            }
+        shouldBeConsumed<Input1> {
```

Comment:
@DariusKlein, I have implemented the changes I mentioned in #560, and improved the ser/de operations. That led us to use the simple and clean DSL again, just FYI. And here are the tests you have written changed a little bit.

Any thoughts?